### PR TITLE
header, menu and routes

### PR DESCRIPTION
**Pull request by rartner**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -23,6 +23,10 @@
...
 
 @keyframes App-logo-spin {
-  from { transform: rotate(0deg); }
-  to { transform: rotate(360deg); }
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
```
Comment (cuzik) [inline src/App.css:31]: nice, my `toc` thanks it (^_^)

File: src/components/Courses.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+      <div>
+        <Header />
+        <p>Cursos</p>
+      </div>
+    );
+  }
+}
+
+export default withStyles(styles)(Courses);
```
Comment (cuzik) [inline src/components/Courses.jsx]: When we don't use `styles` we can do this
```js
export default Courses
```

and remove 
```js
const styles = {};
```

File: src/components/Events.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+      <div>
+        <Header />
+        <p>Eventos</p>
+      </div>
+    );
+  }
+}
+
+export default withStyles(styles)(Events);
```
Comment (cuzik) [inline src/components/Events.jsx]: same ^

File: src/components/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        <Link to="/" className={classes.menuItem}>
+          <ListItem button>
+            <ListItemIcon>
+              <Icon style={styles.udescColor}>home</Icon>
+            </ListItemIcon>
+            <ListItemText primary="Início" />
+          </ListItem>
+        </Link>
+        <Divider />
+        <Link to="/talks" className={classes.menuItem}>
```
Comment (cuzik) [inline src/components/Header.jsx]: We can create `components` one for each link / button, can be done in this same file

File: src/components/Talks.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+      <div>
+        <Header />
+        <p>Talks</p>
+      </div>
+    );
+  }
+}
+
+export default withStyles(styles)(Talks);
```
Comment (cuzik) [inline src/components/Talks.jsx]: `const styles`

**Review by cuzik — CHANGES_REQUESTED** [review]
we can use single quotes default to strings

File: src/App.css (hunk 1 above)
Comment (rartner) [reply src/App.css:31]: :cow:

**Review by cuzik — APPROVED** [review]
:top: :cow:

**Pull request merged**